lib/nunjucks/govuk-prototype-kit/layouts/govuk-branded.html

```diff
@@ -10,7 +10,7 @@
 {% endblock %}
 
 {% block pageTitle %}
-  GOV.UK Prototype Kit
+  {% if title %}{{ title }} - {% endif %}{{ serviceName }} - {% if titleOrganisation %}{{ titleOrganisation }}{% else %}GOV.UK Prototype Kit{% endif %}
```

Comment:
I’m not actually sure if `titleOrganisation` is needed? I've not seen anyone use that format and it’s not mentioned in the [title style guide](https://www.gov.uk/service-manual/design/writing-for-user-interfaces#style) - but maybe you know of some instances?

Reply:
The reason I put that in is because we use ` - GOV.UK Prototype Kit` and your example was ` - GOV.UK`.

I took that to mean that you want titles in your prototype to end with ` - GOV.UK` but I don't think we can recommend that as a default, so in your kit you can set `titleOrganisation = "GOV.UK"` and get the behaviour that was in your PR before I messed with it.

Reply:
Also, I fixed the dash to match what's on the style guide (probably what we were already using).

Reply:
Wouldn’t that still result in `Section name - GOV.UK - GOV.UK Prototype Kit` though? I think we’d want to use `Section name - service name - GOV.UK`, as per the style guide?

Is there any reason not to switch out `GOV.UK Prototype Kit` for `GOV.UK` as the hardcoded bit at the end? The prototypes are password-protected, and otherwise look and feel exactly like a real service, so I don't see any issues around accidentally misleading people?

Reply:
Agree the default should be GOV.UK as per our guidelines